### PR TITLE
Modify rule S1116: Add exception paragraph

### PR DESCRIPTION
**Pull request by francoismora**
Modify rule S1116: Add exception paragraph

### CODE REVIEW

File: rules/S1116/javascript/message.adoc [hunk 1]
```diff
@@ -2,3 +2,19 @@
 
 Remove this extra semicolon.
 
+== Exceptions
+
+This rule does not apply when the semicolon is after a line break and before ``++(++`` or ``++[++``.
```
Comment (saberduck) [inline rules/S1116/javascript/message.adoc:7]: if possible we should explain why we added the exception

**Review by saberduck — CHANGES_REQUESTED** [review]
can you pls add the reason for exception (why)

File: rules/S1116/javascript/message.adoc (hunk 1 above)
Comment (francoismora) [reply rules/S1116/javascript/message.adoc]: Updated

**Comment by francoismora** [comment]
Fixes SonarSource/SonarJS#3250

**Review by saberduck — APPROVED** [review]

**Pull request merged**